carma/launch/guidance.launch.py

```diff
@@ -72,6 +72,9 @@ def generate_launch_description():
 
     plan_delegator_param_file = os.path.join(
         get_package_share_directory('plan_delegator'), 'config/plan_delegator_params.yaml')
+        
+    stop_and_wait_plugin_param_file = os.path.join(
+        get_package_share_directory('stop_and_wait_plugin'), 'config/stop_and_wait_plugin_params.yaml')    
```

Comment:
Please refer to the route_and_following_plugin's comment on where to call this. remove from here.